service/create/error.go

```diff
@@ -38,3 +38,26 @@ var secretsRetrievalFailedError = errgo.New("secrets retrieval failed")
 func IsSecretsRetrievalFailed(err error) bool {
 	return errgo.Cause(err) == secretsRetrievalFailedError
 }
+
+// Validation errors
+
+var workersListEmpty = errgo.New("workers list empty")
```

Comment:
The errors are called `somethingError`. In this case it would be `workersListEmptyError`.